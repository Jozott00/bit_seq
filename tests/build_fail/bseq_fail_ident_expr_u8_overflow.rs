use bit_seq::bseq;

fn main() {
    let var = 0;
    let _: u8 = bseq!(var:9);
}