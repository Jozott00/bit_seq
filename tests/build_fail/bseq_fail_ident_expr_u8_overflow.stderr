error: literal out of range for `u8`
 --> tests/build_fail/bseq_fail_ident_expr_u8_overflow.rs:5:23
  |
5 |     let _: u8 = bseq!(var:9);
  |                       ^^^
  |
  = note: the literal `var` does not fit into the type `u8` whose range is `0..=255`
  = note: `#[deny(overflowing_literals)]` on by default
